.codecov.yml

```diff
@@ -1,7 +1,7 @@
 # Validate with curl --data-binary @.codecov.yml https://codecov.io/validate
 codecov:
   archive:
-    uploads: no
+    uploads: yes
```

Comment:
Probably separate this out into its own change?

Reply:
Will do.

Reply:
Done in #825.